libmultilabel/nn/data_utils.py

```diff
@@ -327,6 +327,13 @@ def get_embedding_weights_from_file(word_dict, embed_file, silent=False, cache=N
 
     embedding_weights = torch.zeros(len(word_dict), embed_size)
 
+    if load_embedding_from_file:
+        # Add UNK embedding
+        # AttentionXML: np.random.uniform(-1.0, 1.0, embed_size)
+        # CAML: np.random.randn(embed_size)
+        unk_vector = torch.randn(embed_size)
+        embedding_weights[word_dict[UNK]] = unk_vector
```

Comment:
Sorry for the confusion.
For loading pre-trained vector like glove, we do not initialize the UNK vector ourselves but simply use the UNK vector provided in glove vector_dict.
On the other hand, for data like MIMIC, we follow CAML to randomly initialize a UNK vector.